Dockerfile

```diff
@@ -13,6 +13,11 @@ RUN cd ~ && openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout wordcamp
 RUN cp ~/wordcamp.crt /etc/ssl/certs/wordcamp.crt
 RUN cp ~/wordcamp.key /etc/ssl/private/wordcamp.key
 
+# Install wkhtmltopdf for camptix invoices. See https://stackoverflow.com/a/38336153/1845153
+RUN curl -L https://github.com/wkhtmltopdf/wkhtmltopdf/releases/download/0.12.4/wkhtmltox-0.12.4_linux-generic-amd64.tar.xz > wkhtmltox.tar.xz
+RUN tar xvf wkhtmltox.tar.xz
+RUN mv wkhtmltox/bin/wkhtmlto* /usr/bin/
+
```

Comment:
The Meta Environment does it a little different:

https://github.com/WordPress/meta-environment/blob/844bef6/wordcamp.test/provision/vvv-init.sh#L23-L29

It'd probably be good to install that latest version via `apt`, since that's probably closer to production. I don't know if the `xvfb-run` stuff is really relevant or not, but it might be.

Reply:
I would rather keep this version locked with whatever we have on production. Also installing with `apt` unfortunately does not comes with few libraries that are needed for this to be run inside docker (qt). 
Also, I'd really avoid a display server just for this if we can, because wkhtmltopdf have its own rendering engine. I think it will be fine without it, but lets add `xvfb` if we see any problems.

Reply:
👍